extensions/teams/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "cwyodts",
+    "name": "tbt",
```

Comment:
What does tbt stand for?

Reply:
Teams Bot Toolkit

Reply:
Is it worth just naming it `teams-bot-toolkit`? Do we need the name shortened?